src/aave-v3/SupplyVaultBase.sol

```diff
@@ -109,9 +109,6 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
     /// PUBLIC ///
 
     /// @notice The amount of assets in the vault.
-    /// @dev The indexes used by this function might not be up-to-date.
-    ///      As a consequence, view functions (like `maxWithdraw`) could underestimate the withdrawable amount.
-    ///      To redeem all their assets, users are encouraged to use the `redeem` function passing their vault tokens balance.
```

Comment:
It is still the case for aave-v3 vaults though.

Reply:
I can re-add it even though we'll need to refactor it anyway (the lens does not exist anymore on Aave V3)

Reply:
Yes, I think that we should keep it for the moment
And I opened #241